homeassistant/components/renson/manifest.json

```diff
@@ -5,5 +5,5 @@
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/renson",
   "iot_class": "local_polling",
-  "requirements": ["renson-endura-delta==1.6.0"]
+  "requirements": ["renson-endura-delta==1.7.1"]
```

Comment:
Please do the version bump in a separate PR

Reply:
I will do it. But my question is why? This is a really small one and it takes more time to create a new PR then reading the change. 

Also when the library has a breaking change in it and only works with the new feauture, do we also need a differnt PR for this? This will break hass if the library is merged without the feature.

Reply:
https://github.com/home-assistant/core/pull/105096

Reply:
In general, we try to have as small PRs as possible, and so we try to separate the version bumps from the feature PRs where possible.

Of course, if that's not possible, as you mentioned, for example with a breaking change, it's absolutely fine for us if it's in a feature/... PR.